### ISSUE TITLE
NIGHT-RUNNERS™ PROLOGUE (2707900)

### ISSUE THREAD

**Issue by ClaudeLib** [ValveSoftware/Proton#7574]
# Compatibility Report
- Name of the game with compatibility issues: NIGHT-RUNNERS™ PROLOGUE
- Steam AppID of the game: 2707900

## System Information
- GPU: AMD Radeon RX 6900 XT
- Video driver version: Mesa 24.0.2-arch1.2
- Kernel version: 6.7.9-arch1-1
- Link to full system information report as Gist: https://gist.github.com/ClaudeLib/5eba8a8f2aa039d7dc835d07cab06006
- Proton version: 8.0-5, 9.0 Beta and Experimental.

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[steam-2707900-8-amdvlk.log](https://github.com/ValveSoftware/Proton/files/14551162/steam-2707900-8-amdvlk.log)
[steam-2707900-8-radv.log](https://github.com/ValveSoftware/Proton/files/14551163/steam-2707900-8-radv.log)
[steam-2707900-9-amdvlk.log](https://github.com/ValveSoftware/Proton/files/14551164/steam-2707900-9-amdvlk.log)
[steam-2707900-9-radv.log](https://github.com/ValveSoftware/Proton/files/14551165/steam-2707900-9-radv.log)
[steam-2707900-e-amdvlk.log](https://github.com/ValveSoftware/Proton/files/14551166/steam-2707900-e-amdvlk.log)
[steam-2707900-e-radv.log](https://github.com/ValveSoftware/Proton/files/14551167/steam-2707900-e-radv.log)

## Symptoms
The game starts fine, but instead of the intro video I only see the test pattern, which softlocks the game.
![20240310213855_1](https://github.com/ValveSoftware/Proton/assets/5920490/99e25e8d-cffb-4832-b668-98d5f73e178f)
By default Steam uses the open-source AMD driver (the Proton logs with it have `amdvlk` in the name), however I also installed the RADV driver `vulkan-radeon lib32-vulkan-radeon` and invoked it by adding `VK_ICD_FILENAMES=/usr/share/vulkan/icd.d/radeon_icd.x86_64.json` to the launch parameters (those logs have `radv` in the name). I tried three versions of Proton, 8.0-5 (`8` in the log name), 9.0 Beta (`9`) and Experimental (`e`), the result is the same across all six combinations.
However, the game seems to run fine on Steam Deck, including video playback: [steam-2707900-deck.log](https://github.com/ValveSoftware/Proton/files/14551168/steam-2707900-deck.log)

## Reproduction
Just run the game, I guess.

**Comment by simifor** [comment]
@ClaudeLib could you try again? It should be working now

**Comment by ClaudeLib** [comment]
Nope, sorry, still no luck. I've updated and rebooted the system several times, restarted the Steam Client in hopes that it would download an update, nothing happened. I begin to think it must be some multimedia codec package which I have to install and which SteamOS has out of the box. Though I forgot to mention that I also tried running the game under HoloISO and ChimeraOS, which mimic SteamOS, and the issue *was* there as well.

**Comment by simifor** [comment]
@ClaudeLib that's weird, it should be working now. Could you try running this in your terminal and see if it starts working? `steam steam://unlockh264/` this is something that should be happening automatically now (wasn't before), but if after enabling h264 manually on your end it doesn't work, please bring new logs for radv and proton stable.

**Comment by ClaudeLib** [comment]
YES! That worked! And the effect remained after client restart. Thank you!
![Screenshot_20240313_214744](https://github.com/ValveSoftware/Proton/assets/5920490/0ecf6788-ccb3-4f5b-8711-1680867f14cc)
![2707900_20240313215105_1](https://github.com/ValveSoftware/Proton/assets/5920490/24f69e47-2a8a-44d9-bca1-47fca7cca55e)

**Comment by kz-n** [comment]
This game on my setup has extremely erratic behaviour (which I think I can attribute to Proton), sometimes it runs at 2fps, sometimes 30 and sometimes > 60fps with no stutters and sometimes it fucks up my system audio making me only hear loud clicks and pops every few seconds until I close it, all of this with seemingly no connection, I tried Proton Experimental and default along with GE Proton and while the latter has higher FPS when the screen is blank, everything most of the time never surpasses 30fps in the garage and 15fps on the road. Changing graphics settings and resolution doesn't seem to impact the FPS.
The game running on the same hardware on Windows has no issues at max settings.
